charts/v3.0.0/csi-driver-nfs/templates/csi-nfs-driverinfo.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
This can be v1, but then min version is 1.18

Reply:
let's do this in next release, 1.18 is too high version